### DISCUSSION TITLE
anyone knows of a VDA5050 compliant version of openTCS ?

### DISCUSSION THREAD

**Discussion by butchamon** [openTCS/opentcs#71] [Q&A] [answered]
Looking into using openTCS to drive several VDA5050 compliant AGV's, does anyone know of such an existing implementation ?

**Comment by swltr** [comment, marked as answer]
@butchamon A first version of a VDA5050 driver for openTCS has just been released - see https://github.com/openTCS/opentcs-commadapter-vda5050.
